.github/workflows/build-and-push-assets.yml

```diff
@@ -99,11 +103,22 @@ jobs:
               .parse(process.env.ENV_VARS)
               .forEach(envVar => core.exportVariable(envVar.name, envVar.value));
 
+      - name: Set compile script
```

Comment:
The indentation is off; there is only one whitespace instead of two, except in line 113.